plugins/services/src/js/services/SidebarFilter.js

```diff
@@ -157,7 +138,9 @@ class SidebarFilter extends mixin(QueryParamsMixin) {
       })
       .map((filterLabel) => {
         let value = filterValues[filterLabel];
-        let checked = selectedNodes.indexOf(value.toString()) > -1;
+        let checked = selectedNodes.indexOf(value.toString()) > -1
+          || selectedNodes.indexOf(value) > -1;
```

Comment:
Actually, we do 👍 because we could get filter values passed down as ints or strings.